src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,24 @@
+import cn from 'classnames';
+import 'bulma/css/bulma.css';
+
+export const Tab = ({ tab, activeTab, changeTabFromTab }) => {
+  return (
+    <button
+      type="button"
+      onClick={() => {
+        changeTabFromTab(tab);
+      }}
+    >
```

Comment:
button can't wrap li in list. and it is redundant here.
set onClick handler for link.

also check and execute changeTabFromTab function only if it is not active tab.
rename changeTabFromTab prop on onTabSelected (it is requirement in description and should help to pass tests.)